src/views/ModelPlan/TaskList/Basics/Milestones/index.tsx

```diff
@@ -235,169 +238,212 @@ const Milestones = () => {
-                    <MINTDatePicker
-                      fieldName="announced"
-                      className="margin-top-4"
-                      id="Milestone-announced"
-                      label={t('annouceModel')}
-                      placeHolder
-                      handleOnBlur={handleOnBlur}
-                      formikValue={values.announced}
-                      value={announced}
-                      error={flatErrors.announced}
-                    />
-                  </div>
-
-                  <legend className="usa-label margin-bottom-neg-2">
-                    {t('applicationPeriod')}
-                  </legend>
-
-                  <div className="datepicker__wrapper">
-                    <MINTDatePicker
-                      fieldName="applicationsStart"
-                      id="Milestone-applicationsStart"
-                      label={t('applicationStartDate')}
-                      boldLabel={false}
-                      placeHolder
-                      handleOnBlur={handleOnBlur}
-                      formikValue={values.applicationsStart}
-                      value={applicationsStart}
-                      error={flatErrors.applicationsStart}
-                      warning={false}
-                    />
-
-                    <MINTDatePicker
-                      fieldName="applicationsEnd"
-                      id="Milestone-applicationsEnd"
-                      label={t('applicationEndDate')}
-                      boldLabel={false}
-                      placeHolder
-                      handleOnBlur={handleOnBlur}
-                      formikValue={values.applicationsEnd}
-                      value={applicationsEnd}
-                      error={flatErrors.applicationsEnd}
-                      warning={false}
-                    />
-                  </div>
+                  <ProcessList className="read-only-model-plan__timeline">
+                    <ProcessListItem className="read-only-model-plan__timeline__list-item">
+                      <ProcessListHeading
+                        type="p"
+                        className="font-body-sm line-height-sans-4"
+                      >
+                        <div className="datepicker__wrapper">
+                          <MINTDatePicker
+                            className="margin-top-0"
+                            fieldName="completeICIP"
+                            id="Milestone-completeICIP"
+                            label={t('completeICIP')}
+                            placeHolder
+                            handleOnBlur={handleOnBlur}
+                            formikValue={values.completeICIP}
+                            value={completeICIP}
+                            error={flatErrors.completeICIP}
+                          />
+                        </div>
+                      </ProcessListHeading>
+                    </ProcessListItem>
```

Comment:
![image](https://user-images.githubusercontent.com/2349702/215577086-1deedd17-62cc-4854-862e-46e6bdab9ba9.png)

Reply:
The widths of the date picker is too small and causing the year to be scrunched to a new line.